plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/DefaultLocationProvider.kt

```diff
@@ -41,12 +40,45 @@ internal class LocationProviderImpl(
   private var handler: Handler? = null
   private lateinit var runnable: Runnable
   private var updateDelay = INIT_UPDATE_DELAY
+  @VisibleForTesting(otherwise = PRIVATE)
+  internal val locationEngineCallback = object : LocationEngineCallback<LocationEngineResult> {
```

Comment:
@pengdev isn't your PR here interfering with https://github.com/mapbox/mapbox-maps-android/pull/1093? Perhaps merge that one first and get clearer delta here?

Reply:
As discussed with @Chaoba , we will merge this PR first and rebase in #1093